### PR TITLE
ci: add .github folder

### PR DESCRIPTION
**Pull request by Soaib024**
### Release required?

- [x] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Primary owner should be listed first in list of global owners, followed by any secondary owners
+
```
Comment (ocofaigh) [inline .github/CODEOWNERS:2]: Can you add codeowners to this? Suggest to use same codeowners as our internal vpc module

File: .github/settings.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+# repo-specific settings
+#
+repository:
+  # See https://terraform-ibm-modules.github.io/documentation/#/implementation-guidelines?id=module-names-and-descriptions
+  name: "terraform-ibm-vpc"
+
+  # The description is displayed under the repository name on the
+  # organization page and in the 'About' section of the repository.
+
+  description: "Provisions and configures IBM Cloud"
```
Comment (ocofaigh) [inline .github/settings.yml:19]: this makes no sense - did you mean to say "Provisions and configures IBM Cloud VPC resources"?

**Review by ocofaigh — CHANGES_REQUESTED** [review]
Left some comments - Dont add the pipeline yet - it wont pass until we have tests

File: .github/CODEOWNERS (hunk 1 above)
Comment (Soaib024) [reply .github/CODEOWNERS]: added

File: .github/settings.yml (hunk 2 above)
Comment (Soaib024) [reply .github/settings.yml]: 😅 actually yes, added VPC resources

**Review by ocofaigh — APPROVED** [review]

**Pull request merged**